css/custom.css

```diff
@@ -1,43 +1,138 @@
-.container {
+@font-face {
+    font-family: comicfont;
+    src: url(../assets/fonts/BADABB__.TTF);
+}
+
+@font-face {
+    font-family: button-comic-font;
+    src: url(../assets/fonts/ObelixPro-Broken-cyr.ttf);
+}
+
+body {
+  background-color: #FDD476;
+}
+
+h1 {
+  font-size: 8em;
+}
+
+h3 {
+  font-size: 2em;
+}
+
+.title {
+  font-family: comicfont, Helvetica;
+}
+
+#communication { 
+  font-family: button-comic-font;
+}
+.container-fluid {
   margin:auto;
-  min-width:1000px;
+  min-width: 800px;
+
 }
 
 #game-screen {
-  margin-left:100px;
+  background-color: rgba(0,0,0,0.4);
+  display: inline-block;
+  border-radius: 15px;
+  padding-right: 80px;
+  margin-left: 25px;
 }
 
-
 .cup {
-  min-width: 100px;
-  min-height: 100px;
+<<<<<<< HEAD
```

Comment:
please remove this `+<<<<<<< HEAD`, it comes from a merge conflict... and shouldn't be here